templates/default/engage-nginx-proxy-conf.erb

```diff
@@ -1,6 +1,7 @@
 server {
   listen 80 default_server;
   listen [::]:80 default_server ipv6only=on;
+  directio 1M;
```

Comment:
This optimization exists to avoid a lot of buffer cache churn.  Unless you have a machine with an ungodly amount of RAM or a situation where it serves the same large file over and over again, the buffer cache is going to have a very low hit rate for large video files.

I set it to 1M so that everything under that - the JPG thumbnails - would be stored in the buffer cache but none of the video files would. 

In reality, if we've tuned cloudformation correctly we shouldn't need to care about this at all.  If the setting is wrong, it's just going to lead to more files being served directly off the filesystem.

I went back-and-forth on this but basically every nginx tutorial about tuning large files suggested managing the directio setting. It seems pretty low risk especially since this is just going to be a CDN seed server.